backend/src/helpers/helpers.js

```diff
@@ -0,0 +1,41 @@
+// Priorities are set from the lowest to the highest.
+const statePriorities = [ 'away', 'viewing', 'commenting', 'editing', 'merging' ];
```

Comment:
I prefer to use a UPPER_CASE for const - `const STATE_PRIORITES` ;) But it is up to you

Reply:
IMO it looks better too. I've changed it to UPPER_CASE.